mmv1/third_party/terraform/resources/resource_dns_record_set.go

```diff
@@ -620,3 +621,13 @@ func flattenDnsRecordSetRoutingPolicyGEO(geo *dns.RRSetRoutingPolicyGeoPolicy) [
 	}
 	return ris
 }
+
+func validateRecordNameTrailingDot(v interface{}, k string) (warnings []string, errors []error) {
+	value := v.(string)
+	last1 := value[len(value)-1:]
```

Comment:
there is potential to crash here if we are out of bounds. Could you add empty string as a edge case in your test suite?

https://go.dev/play/p/At9a9rVQlRL

Reply:
Will do @ScottSuarez